libs/pages/elewa/news/src/lib/components/button-plain/button-plain.component.html

```diff
@@ -0,0 +1 @@
+<button (click)="clickHandler()">{{innerText}}</button>
```

Comment:
The function of this button is very simple, does not need a separate component. Too abstract